ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -287,8 +332,11 @@ class AlertTabs extends Component {
           <TabList customClass="config-endpoint--tabs">
             {_.reduce(
               configSections,
-              (acc, _cur, k) =>
-                supportedConfigs[k]
+              (acc, _cur, k) => {
+                return supportedConfigs[k] &&
```

Comment:
I might argue that at the point we have an && on a ternary is when it should be extracted. What do you think?

Reply:
I will move the top part out to be a function